mir-it/src/test/java/org/mycore/mir/it/controller/MIREditorController.java

```diff
@@ -30,6 +31,14 @@ protected void setTextAreaText(String childElementName, String text) {
         driver.waitAndFindElement(By.xpath(".//textarea[contains(@name,'" + childElementName + "')]")).sendKeys(text);
     }
 
+    protected void setHTMLAreaText(String childElementName, String text) {
+        final WebElement iframe = driver.waitAndFindElement(By.xpath(".//div[contains(@id,'" + childElementName + "') and contains(@class, 'cke')]//iframe"));
```

Comment:
Codacy found an issue: [Line is longer than 120 characters (found 158).](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=5834770)